Makefile

```diff
@@ -22,12 +22,12 @@ REAL_CLONE_URL = $(if $(CLONE_URL),$(CLONE_URL),$(DEFAULT_CLONE_URL))
 
 # Run code quality checks
 style_check: clean
-	pip install -U pip ruff
+	python.exe -m pip install -U pip ruff
```

Comment:
Why is the python.exe required here?

Reply:
my mistake sorry. I left it there after my local test on windows machine. I fix it